scripts/release/generate-changelog.js

```diff
@@ -12,9 +12,9 @@ const { modifyFile } = require('../lib/files-utils')
 
 const CHANGELOG_FILE = 'CHANGELOG.md'
 const CONTRIBUTING_FILE = 'CONTRIBUTING.md'
-const EMOJI_PRIORITY = ['💥', '✨', '🐛', '⚗️', '♻️']
-const START_WITH_EMOJI = /^\p{Emoji_Presentation}/u
-
+const PUBLIC_EMOJI_PRIORITY = ['💥', '✨', '🐛', '⚡️', '📝', '⚗️']
+const INTERNAL_EMOJI_PRIORITY = ['👷', '🎨', '🧪', '✅', '👌', '♻️', '🔧', '📄', '🔇', '🔊', '📦']
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🥜 nitpick:** ‏I would reorder the table to put the emojis of the same "nature" close to one another:
```
['👷', '🔧', '📦', // build conf‏
'♻️', '🎨', // refactoring
'🧪', '✅',  // tests
'🔇', '🔊', // telemetry
'👌', '📄']
```